packages/integrations/sitemap/src/index.ts

```diff
@@ -168,13 +168,15 @@ const createPlugin = (options?: SitemapOptions): AstroIntegration => {
 						}
 					}
 					const destDir = fileURLToPath(dir);
+					const xslURL = opts.xslURL ? new URL(opts.xslURL, finalSiteUrl).href : undefined;
```

Comment:
Quoting docs PR:

> The URL can be either relative or absolute. If it is relative, it will be resolved relative to the `site` URL.

I don't think this logic is covered here. Should we update the docs to reflech this behavior or update the behavior to match docs? (the latter makes more sense imo)

Reply:
I think relative and absolute works here? Or do you mean the fact that `finalSiteUrl` isn't exactly relative to the `site` URL, but `site` + `base`?

Reply:
Ah maybe I misunderstood "absolute"? I understood it like this:
- relative: "/sitemap.xsl" => "https://example.com/sitemap.xsl"
- absolute: "https://foo.bar/baz.xsl" => "https://foo.bar/baz.xsl"

Reply:
That's how I understand it too, but isn't it already working that way?

Reply:
Indeed it already works :sweat_smile: TIL